climada_petals/hazard/test/test_coastal_flood.py

```diff
@@ -0,0 +1,73 @@
+        
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Tests on Coastal Flood Hazard"""
+
+import unittest
+import requests
+import numpy as np
+from climada_petals.hazard.coastal_flood import AQUEDUCT_SOURCE_LINK
+
+RCPS = ['historical', '45', '85']
+TARGET_YEARS = ['hist', '2030', '2050', '2080']
+RETURN_PERIODS = [2, 5, 10, 25, 50, 100, 250, 500, 1000]
+SUBSIDENCE = ['nosub', 'wtsub']
+PERCENTILES = ['5', '50', '95']
+
+def rcp_name(rcp):
+    return f"rcp{rcp[0]}p{rcp[1]}" if rcp in ['45', '85'] else rcp
+
+def rp_name(return_period):
+    return f"{str(return_period).zfill(4)}"
+
+def perc_name(percentile):
+    return f"0_perc_{percentile.zfill(2)}" if percentile in ['05', '50'] else '0'
```

Comment:
Why not define these methods in the main class? Would this not make more sense?

Reply:
I removed the use of these functions altogether